scripts/typescript_checks.py

```diff
@@ -82,12 +835,91 @@ def compile_and_check_typescript(config_path: str) -> None:
     assert process.stdout is not None
     error_messages = list(iter(process.stdout.readline, ''))
 
-    if error_messages:
-        print('Errors found during compilation\n')
-        print('\n'.join(error_messages))
-        sys.exit(1)
+    if config_path == STRICT_TSCONFIG_FILEPATH:
```

Comment:
Should the content of this if-block be separated into a function? Otherwise the `else` for this `if` is so far away.

Reply:
Done